__tests__/app.test.js

```diff
@@ -26,3 +23,28 @@ describe("GET /api", () => {
       });
   });
 });
+
+describe("GET /api/topics", () => {
+  test("200: returns all the topics with the desired categories", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
+        body.forEach((topic) => {
+          expect(body).toHaveLength(3);
+          expect(topic).toMatchObject({
+            slug: expect.any(String),
+            description: expect.any(String),
+          });
+        });
+      });
+  });
+  test("404: responds with a 'not found' message", () => {
+    return request(app)
+      .get("/api/route")
+      .expect(404)
+      .then(({ body }) => {
+        expect(body).toEqual({ message: "not found" });
```

Comment:
Nice error handling test 🥇 We're looking for a new object with a message key here as well, nice one! 🥇